### ISSUE TITLE
Method _init conflicts with an existing Vue instance method

### ISSUE THREAD

**Issue by xralphack** [zhanziyang/vue-croppa#32]
I used croppa in component

```
	import Croppa from 'vue-croppa';
	import 'vue-croppa/dist/vue-croppa.css';

	export default {
		components: {
			croppa: Croppa.component
		},
```

There is a warning.

[Vue warn]: Method "_init" conflicts with an existing Vue instance method.

**Comment by miljko1984** [comment]
Same here.

**Comment by zhanziyang** [comment]
Fixed in v1.0.3. Thanks!

**Comment by miljko1984** [comment]
Thank you!
